### ISSUE TITLE
Not loaded

### ISSUE THREAD

**Issue by AndroidHackPro** [AndroidHackPro/LiveSigns-PHP-7#2]
@killer549 Now the server run with the plugin, but when I write the command `/plugins` appears don't work or loaded

**Comment by ahmedhozny** [comment]
did u get any message on the CMD ?

**Comment by AndroidHackPro** [comment]
All plugins appears in green color and the LiveSigns name appears in red color

**Comment by ahmedhozny** [comment]
yeah i know, when the server load already, if there's an error u will get the cause of the error in the cmd

**Comment by AndroidHackPro** [comment]
Here is! 

Server thread/CRITICAL ClassNotFoundException: "Class aliuly\livesigns\common\mc not found" (EXCEPTION) in "/src/spl/BaseClassLoader" at line 144

**Comment by DrasticInsanity** [comment]
Have you tried to replace the Main.php in the plugin that us on BukGet through Virtual Gladiators because that claims to have been updated to API 2.0 and that might work to get it loaded? I can leave a link to the .phar for it

**Comment by DrasticInsanity** [comment]
Does it not loading have anything to do with Genysis?

**Comment by AndroidHackPro** [comment]
What you recommend @DrasticInsanity?

**Comment by DrasticInsanity** [comment]
Well you fixed the php so the server knows that it's there and it recognizes the plugin because the server doesn't crash but it doesn't have an updated API to run it I would suggest maybe another plugin that creates an API for all of your plugins you would have to look into it but I know there are plugins on Bukkit that does this and there may be a ported version for the PE version. I know that the server ConflictPE figured out someway to fix this you would have to talk to their developer his Twitter is @ItzDaDobby

**Comment by AndroidHackPro** [comment]
@killer549 please fix it because I really need this plugin 

 `Server thread/CRITICAL ClassNotFoundException: "Class aliuly\livesigns\common\mc not found" (EXCEPTION) in "/src/spl/BaseClassLoader" at line 144`

**Comment by AndroidHackPro** [comment]
@DrasticInsanity you know how fix this issue?

**Comment by LewBr** [comment]
This is because on the plugin doesnt have the file and paste common, mc!

**Comment by DrasticInsanity** [comment]
So did you fix this?
